### PR TITLE
Change document title

### PR DESCRIPTION
**Pull request by selankon**
Fixes https://github.com/vocdoni/ui-scaffold/issues/614

It changes the document title on certain pages using the `useDocumentTitle` hook.

When the hook is unmounted it goes back to the default page title

### CODE REVIEW

**Review by elboletaire — APPROVED** [review]

File: src/elements/Process.tsx [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+}
 
 const Process = () => {
   const election = useLoaderData() as PublishedElection
 
   return (
     <OrganizationProvider id={election.organizationId}>
       <ElectionProvider election={election} ConnectButton={ConnectButton} fetchCensus autoUpdate>
-        <ProcessView />
+        <SetProcess />
```
Comment (elboletaire) [inline src/elements/Process.tsx:20]: Terrible name 😅

Comment (selankon) [reply src/elements/Process.tsx]: True! 🙈 dbf3e00

**Review by elboletaire — CHANGES_REQUESTED** [review]
I see a problem here... navigating through pages maintains one of the first titles set (?)

To reproduce:
- Go to your organization, you'll se the title change
- Go to the home page, the title remains with the org name

**Comment by selankon** [comment]
FTR if we would like to centraliza all on an only one hook/component that just change the title, and dinamically check if is inside an election context or organization context to set the title, i did this approach which is not finished but can be used as inspiration if we need to do it on the future

```ts
export function useDocumentTitle(title?: string) {
  const defaultTitleRef = useRef(document.title)
  const electionCtx = useContext(ElectionContext)
  const orgCtx = useContext(OrganizationContext)

  useEffect(() => {
    const isOnvote = import.meta.env.theme === 'onvote'
    let prepend = 'Vocdoni'
    // Vocdoni - The voice of digital voting
    // Anonymous Gasless and Modular voting for Web3
    let title = 'The voice of digital voting'
    if (isOnvote) {
      title = defaultTitleRef.current
      prepend = 'OnVote'
    }
    if (electionCtx && electionCtx.election instanceof PublishedElection) {
      title = electionCtx.election?.title.default
    } else if (orgCtx && orgCtx.organization) {
      title = orgCtx.organization?.account.name.default ?? orgCtx.organization?.address ?? ''
    }

    document.title = `${prepend} - ${title}`
    return () => {
      document.title = defaultTitleRef.current
    }
  }, [electionCtx, orgCtx, title])
}
```

**Comment by elboletaire** [comment]
> I see a problem here... navigating through pages maintains one of the first titles set (?)
> 
> To reproduce:
> 
>     * Go to your organization, you'll se the title change
> 
>     * Access a process, the title changes again
> 
>     * Go to the home page, the title remains with the election name (?)

I can still reproduce this 😕

**Review by elboletaire — APPROVED** [review]

**Pull request merged**